### ISSUE TITLE
Failed to parse onnx model when Softmax takes 1d-vector as input

### ISSUE THREAD

**Issue by qwaker00** [onnx/onnx-tensorrt#898]
## Description
Execution of ```trtexec``` fails while parsing model with Softmax 1d-vector input.
Error message:
```
[02/01/2023-06:57:28] [E] [TRT] ModelImporter.cpp:732: ERROR: ModelImporter.cpp:185 In function parseGraph:
[6] Invalid Node - /Softmax
/Softmax: at least 2 dimensions are required for input.
```
But [onnx doc](https://onnx.ai/onnx/operators/onnx__Softmax.html#softmax-13) allows 1d input starting from opset 13. 

## Steps To Reproduce
1. Export onnx model with torch:
```
python export.py
```
Where ```export.py``` is:
```python
import torch
import torch.nn as nn
import torch.nn.functional as F

class Model(nn.Module):
    def __init__(self):
        super().__init__()
        self.weights = nn.Parameter(torch.randn(10))

    def forward(self, x):
        y = F.softmax(self.weights * x, dim=0)
        return y

model = Model()
x = torch.rand(10)
y = model(x)
torch.onnx.export(
    model, (x, ), "model.onnx", verbose=True,
    input_names=["x"], output_names=["y"],
    opset_version=17,
)
```
2. Run trtexec:
```
/usr/src/tensorrt/bin/trtexec --onnx=./model.onnx --saveEngine=model.trt
```

[Full stderr](https://gist.githubusercontent.com/qwaker00/bf3ea03961c4eba73be611481b4308be/raw/f575e1557f473484f15a4861ee1486de33761b72/err) [Full stdout](https://gist.githubusercontent.com/qwaker00/f8912614fa20069142a8cea192f39581/raw/ded7780904045022ff24a967aeb9a12201011cdc/out)

## Environment

**TensorRT Version**:  8.5.1.7-1+cuda11.8
**ONNX-TensorRT Version / Branch**:
**GPU Type**: A100-SXM4-40GB
**Nvidia Driver Version**: 470.42.01
**CUDA Version**: 11.4
**CUDNN Version**: 
**Operating System + Version**: Ubuntu 18.04.6 LTS
**Python Version (if applicable)**: 3.8.0
**TensorFlow + TF2ONNX Version (if applicable)**: 
**PyTorch Version (if applicable)**:  1.13.0+cu117
**Baremetal or Container (if container which image + tag)**:

**Comment by zhenhuaw-me** [comment]
@qwaker00 Thank you for reporting this issue. TensorRT currently requires at least 2D for Softmax input. We will evaluate the 1D requirement and enable in the future release.

Before that happens, could you please try to workaround this issue by changing the subgraph from `1D - Softmax - 1D` to `1D - Unsqueeze - 2D - Softmax - 2D - Squeeze - 1D`? Thanks!
